### PR TITLE
Issue 1711: Fix org.vafer:jdeb plugin

### PR DESCRIPTION
**Pull request by valiobojinov**
# Pull Request Description

This pull request closes strongbox/strongbox#1711 .

# Acceptance Test

* [x] Building the code with `mvn clean install -Dintegration.tests` still works.
* [x] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [x] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [x] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.

# Questions

* Does this pull request break backward compatibility? 
  * [ ] Yes
  * [x] No

* Does this pull request require other pull requests to be merged first? 
  * [ ] Yes, please see #...
  * [x] No

* Does this require an update of the documentation?
  * [ ] Yes, please see strongbox/strongbox-docs#{PR_NUMBER}
  * [x] No

### CODE REVIEW

**Comment by valiobojinov** [comment]
I've had some issues with the web integration tests failing due to port 9001 not being free, however, few hours later they passed without any failures has that happened before?

**Comment by carlspring** [comment]
We don't use that port for anything. Were you perhaps running remote debugging, or something else on that port? :)

File: pom.xml [hunk 1]
```diff
@@ -103,6 +103,7 @@
         <version.semver>0.9.34-SNAPSHOT</version.semver>
         <version.springfox>2.9.2</version.springfox>
         <version.redline>1.2.9</version.redline>
+        <version.jdeb>1.8</version.jdeb>
```
Comment (carlspring) [inline pom.xml]: We don't need a property for this. Version properties should only be defined for dependencies and not plugins.

**Comment by carlspring** [comment]
Would you mind signing the ICLA, as described in the [Contributing](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) page?

File: pom.xml (hunk 1 above)
Comment (valiobojinov) [reply pom.xml]: I guess there are some plugins with versions that are bound to other dependencies, aren't they? I've spotted mix version convention handling, thus defined that one, I'll have it fixed.

**Comment by valiobojinov** [comment]
> We don't use that port for anything. Were you perhaps running remote debugging, or something else on that port? :)

Initially I thought so, as this 9001 does ring a bell for me, I checked with **lsof -i :9001** the process was a java one, I had it killed and re-run the integration tests once again and it failed two times in a row, I still wonder why it would attempt to open when it isn't used. Either way will give it another go tomorrow when modify the pom

**Comment by carlspring** [comment]
Yeah, I've seen those. We should clean them up as well (separately), but, for the time-being, let's just handle this particular case.

**Comment by steve-todorov** [comment]
Hmm, I have a vague memory that we have some services start at randomly allocated ports when the tests start. I believe it was the embedded ldap server or something like that. Maybe some of those services are overlapping with the debugger ports?

File: pom.xml (hunk 1 above)
Comment (valiobojinov) [reply pom.xml]: done

**Comment by carlspring** [comment]
Thank you! Congratulations on your first pull request! :)

**Pull request merged**